.gitignore

```diff
@@ -1,4 +1,4 @@
-
+.idea
```

Comment:
We should probably exclude files/directories not generated by the code in this repository from `.gitignore`. Maintaining own `.gitignore_global` for OS/IDE-specific local files is best practice.